### PR TITLE
2 packages from yasuo-ozu/satysfi-xpath at 0.2.0

### PR DESCRIPTION
**Pull request by yasuo-ozu**
This pull-request concerns:
-`satysfi-xpath.0.2.0`: Advanced path algorithms in SATySFi
-`satysfi-xpath-doc.0.2.0`: Docs of advanced path algorithms in SATySFi

---
* Homepage: https://github.com/yasuo-ozu/satysfi-xpath
* Source repo: git+https://github.com/yasuo-ozu/satysfi-xpath.git
* Bug tracker: https://github.com/yasuo-ozu/satysfi-xpath/issues

---
:camel: Pull-request generated by opam-publish v2.1.0
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-xpath-doc.0.2.0 satysfi-xpath.0.2.0
- [x] Add to snapshot `snapshot-develop--1` :: satysfi-xpath-doc.0.2.0 satysfi-xpath.0.2.0
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-5` :: satysfi-xpath-doc.0.2.0 satysfi-xpath.0.2.0
- [x] Add to snapshot `snapshot-stable-0-0-6` :: satysfi-xpath-doc.0.2.0 satysfi-xpath.0.2.0
- [x] Add to snapshot `snapshot-stable-0-0-6--1` :: satysfi-xpath-doc.0.2.0 satysfi-xpath.0.2.0

### CODE REVIEW

File: packages/satysfi-xpath-doc/satysfi-xpath-doc.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+authors: "Yasuo Ozu <yasuo@ozu.email>"
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/yasuo-ozu/satysfi-xpath"
+bug-reports: "https://github.com/yasuo-ozu/satysfi-xpath/issues"
+dev-repo: "git+https://github.com/yasuo-ozu/satysfi-xpath.git"
+
+depends: [
+  "satysfi" {>= "0.0.5" & < "0.0.7"}
+  "satysfi-dist"
+  "satysfi-xpath"
```
Comment (na4zagin3) [inline packages/satysfi-xpath-doc/satysfi-xpath-doc.0.2.0/opam:15]:
```suggestion
  "satysfi-xpath" {= "%{version}%"}
```

**Comment by yasuo-ozu** [comment]
Thanks for revising.

**Comment by na4zagin3** [comment]
let me rerun ci script by reopening the ticket

**Comment by na4zagin3** [comment]
Thank you!

**Pull request merged**